src/background.ts

```diff
@@ -203,6 +203,69 @@ async function installVvppEngine(vvppPath: string) {
   }
 }
 
+/**
+ * 危険性を案内してからVVPPエンジンをインストールする。
+ * FIXME: こちらで案内せず、GUIでのインストール側に合流させる
+ */
+async function installVvppEngineWithWarning({
+  vvppPath,
+  restartNeeded,
+}: {
+  vvppPath: string;
+  restartNeeded: boolean;
+}) {
+  const result = dialog.showMessageBoxSync(win, {
+    type: "warning",
+    title: "エンジン追加の確認",
+    message: `この操作はコンピュータに損害を与える可能性があります。エンジンの配布元が信頼できない場合は追加しないでください。`,
+    buttons: ["追加", "キャンセル"],
+    noLink: true,
+    cancelId: 1,
+  });
+  if (result == 1) {
+    return;
+  }
+
+  await installVvppEngine(vvppPath);
+
+  if (restartNeeded) {
+    dialog
+      .showMessageBox(win, {
+        type: "info",
+        title: "再起動が必要です",
+        message:
+          "VVPPファイルを読み込みました。反映には再起動が必要です。今すぐ再起動しますか？",
+        buttons: ["再起動", "キャンセル"],
+        noLink: true,
+        cancelId: 1,
+      })
+      .then((result) => {
+        if (result.response === 0) {
+          appState.willRestart = true;
+          app.quit();
+        }
+      });
+  }
+}
+
+/**
+ * マルチエンジン機能が有効だった場合はtrueを返す。
+ * 無効だった場合はダイアログを表示してfalseを返す。
+ */
+function checkMultiEngineEnabled(): boolean {
+  const enabled = store.get("experimentalSetting").enableMultiEngine;
+  if (!enabled) {
+    dialog.showMessageBoxSync(win, {
+      type: "info",
+      title: "マルチエンジン機能が無効です",
+      message: `マルチエンジン機能が無効です。有効にするには設定で有効にしてください。`,
```

Comment:
```suggestion
      message: `マルチエンジン機能が無効です。vvppファイルを使用するには設定からマルチエンジン機能を有効にしてください。`,
```

ちょっとだけ丁寧にしてみました、ここら辺は正直好みだと思うので、取り込まなくても大丈夫だと思います！